test-case/check-pause-release-suspend-resume.sh

```diff
@@ -76,8 +76,11 @@ OPT_HAS_ARG['i']=1             OPT_VAL['i']='500'
 OPT_NAME['s']='sof-logger'   OPT_DESC['s']="Open sof-logger trace the data will store at $LOG_ROOT"
 OPT_HAS_ARG['s']=0             OPT_VAL['s']=1
 
-OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg path, default is environment variable: TPLG'
-OPT_HAS_ARG['t']=1         OPT_VAL['t']="$TPLG"
+OPT_NAME['t']='tplg'         OPT_DESC['t']='tplg path, default is environment variable: TPLG'
+OPT_HAS_ARG['t']=1             OPT_VAL['t']="$TPLG"
+
+OPT_NAME['T']='type'         OPT_DESC['T']="suspend/resume type from /sys/power/mem_sleep"
+OPT_HAS_ARG['T']=1             OPT_VAL['T']=""
```

Comment:
I was not able to understand what the option means. the 'from' part threw me off, did you mean instead that it would set the suspend/resume type by writing to sysfs?

Reply:
This option is merely passed  passed-through to `test-case/check-suspend-resume.sh` and the description is merely copied from there (which does not mean it cannot be improved)